src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+@import "../utils/variables";
+
+.main {
```

Comment:
> it does not belong to card. Maybe will be better to create separate file for such blocks?

idk where actually it should be, because its part of my container-card